### PR TITLE
tests, backend-tests: move create organization test from integration to backend integration

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issues: https://tracker.mender.io/browse/QA-32

### CODE REVIEW

File: testutils/api/tenantadm.py [hunk 1]
```diff
@@ -14,9 +14,11 @@
```
Comment (lluiscampos) [inline testutils/api/tenantadm.py]: Once you are at it, please update copyright year here also.

File: backend-tests/tests/test_create_org.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    tenantadm_cli = CliTenantadm()
+    tenantadm_cli.migrate()
+    yield clean_mongo
+
+class TestCreateOrganizationEnterprise:
+    def test_success(self, clean_migrated_mongo):
+        tc = ApiClient(tenantadm.URL_MANAGEMENT)
+        uc = ApiClient(useradm.URL_MGMT)
+
+        logging.info("Starting TestCreateOrganization")
```
[the remark this answers is not in the document]
Comment (kjaskiewiczz) [reply backend-tests/tests/test_create_org.py]: fixed

File: backend-tests/tests/test_create_org.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                    useradm.URL_LOGIN,
+                    auth=("some.user@example.com", "asdfqwer1234"))
+        assert r.status_code == 200
+
+    def test_duplicate_organization_name(self, clean_migrated_mongo):
+        payload = {"request_id": "123456", "organization": "tenant-foo", "email":"some.user@example.com", "password": "asdfqwer1234", "g-recaptcha-response": "foobar"}
+        rsp = tc.post(tenantadm.URL_MANAGEMENT_TENANTS, data=payload)
+        assert rsp.status_code == 202
+
+        payload = {"request_id": "123456", "organization": "tenant-foo", "email":"some.user1@example.com", "password": "asdfqwer1234", "g-recaptcha-response": "foobar"}
```
[the remark this answers is not in the document]
Comment (kjaskiewiczz) [reply backend-tests/tests/test_create_org.py]: I think it's not but it's really small change so I've changed request_ids

File: testutils/api/tenantadm.py [hunk 4]
```diff
@@ -14,9 +14,11 @@
 
 URL = 'http://mender-tenantadm:8080/api'
 URL_INTERNAL = URL + '/internal/v1/tenantadm'
+URL_MANAGEMENT = URL + '/management/v1/tenantadm'
```
[the remark this answers is not in the document]
Comment (kjaskiewiczz) [reply testutils/api/tenantadm.py]: done

File: testutils/api/tenantadm.py (hunk 1 above)
Comment (kjaskiewiczz) [reply testutils/api/tenantadm.py]: done

**Pull request merged**